kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -189,6 +189,12 @@ data:
     # [agent_settings.proxy_config]
     #    ignore_proxy_settings = "true"  # if this is not applied, default value is false
 
+    #Configuration settings for the waittime for the network listeners to be available
+    [agent_settings.network_listener_waittime] 
```

Comment:
1. Make sure to comment out these with default values in the final config map.
2.  Add the comment on each port so that its easy for customer and CSS to understand what this port used for